app/waf/src/controllers/common.controller.ts

```diff
@@ -0,0 +1,296 @@
+import {ParameterObject, ParameterLocation} from '@loopback/openapi-v3-types';
+
+const adcCreateRequestProperties = {
+  id: {
+    type: 'string',
+  },
+  name: {
```

Comment:
Defining the whole bunch of things is boring since they are very similar to our model definition.
I'm wondering if we can generate them from our model definition, typescript support reflection. 
How about using the following possible code core to do the schema definition: 

```  
let props = Adc.definition.properties;
for(let p of ['id', 'name', 'createdAt', 'UpdatedAt']) {
    delete props[p];
}

adcCreateRequestProperties = props;
```

Reply:
Let me think

Reply:
Done.